workflow/config/empirical_pred_config.yaml

```diff
@@ -0,0 +1,50 @@
+vcf_path: "empirical_prediction/test_data/test.vcf"
```

Comment:
Hmm, the ideal thing would be to have a single config that's used for both training and prediction, with additional arguments passed in via the command line (or in an optional section in the "training config" -- it would be ignored for the training pipeline, and this pipeline would error out if that is missing.

But, maybe that consolidation can happen later, after this new workflow solidifies a bit?